lightning/backend/tidb.go

```diff
@@ -42,14 +43,15 @@ type tidbEncoder struct {
 
 type tidbBackend struct {
 	db *sql.DB
+	OnDuplicate string
```

Comment:
Does any code outside this module needs to access this field? If not make the field private.

```suggestion
	onDuplicate string
```

(also, `go fmt` this file).